src/combineReducers.js

```diff
@@ -115,13 +115,13 @@ export default function combineReducers(reducers) {
       finalReducers[key] = reducers[key]
     }
   }
-  var finalReducerKeys = Object.keys(finalReducers)
+  const finalReducerKeys = Object.keys(finalReducers)
 
   if (NODE_ENV !== 'production') {
     var unexpectedKeyCache = {}
```

Comment:
On second thought, the `let` fix should be fine, because if it's using variable hoisting as it is. Then defining let before the block is no problem. Let me know what you guys think.